libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -32,9 +33,7 @@ export class BookSearchComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    this.store.select(getAllBooks).subscribe(books => {
-      this.books = books;
-    });
+    this.books$ = this.store.select(getAllBooks);
```

Comment:
You could just assign this on line 20 and get rid of the `ngOnInit` entirely...

Reply:
Removed OnInIt life cycle hook and moved logic to the declaration part.